wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -682,7 +683,8 @@ impl CommandHandler {
             WalletCommand::SendToAddress { address, amount } => {
                 let amount = parse_coin_amount(chain_config, &amount)?;
                 let address = parse_address(chain_config, &address)?;
-                controller_opt
+                // TODO: Take status into account
+                let _status = controller_opt
```

Comment:
@OBorce Please make sure you make use of this too once it's merged. There are a 3 in total.